src/current/v23.2/log-sql-statistics-to-datadog.md

```diff
@@ -71,6 +71,13 @@ Enable the [`sql.telemetry.query_sampling.enabled` cluster setting]({% link {{ p
 SET CLUSTER SETTING sql.telemetry.query_sampling.enabled = true;
 ~~~
 
+Set the [`sql.telemetry.query_sampling.mode` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) to `statement` so that `sampled_query` events are emitted (`sampled_transaction` events will not be emitted):
```

Comment:
strongly recommend adding an anchor link directly to the specific cluster setting 

however I just looked at the [cluster settings page](https://www.cockroachlabs.com/docs/v23.2/cluster-settings) and am not seeing this setting there. is the setting new?

ok after looking into this, it does not appear to be a public cluster setting

it only appears when you type `SHOW ALL CLUSTER SETTINGS` (not `SHOW CLUSTER SETTINGS`)
the `f` in the 4th column means "is it a public setting?" is false

```
 sql.telemetry.query_sampling.mode                                               | statement                               | e            |   f    | the execution level used for telemetry sampling. If set to 'statement', events are sampled at the statement execution level. If set to 'transaction', events are sampled at the txn execution level, i.e. all statements for a txn will be logged and are counted together as one sampled event (events are still emitted one per statement) [statement = 0, transaction = 1]           
 ```
 
IIUC in the code, cluster settings that are meant to be public need to have a `settings.WithPublic` line [like this](https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/sqlstats/cluster_settings.go#L43)

recommend following up with engineering to make sure this cluster setting is set to public, and then you can do a direct link to it in the docs

doesn't have to block this PR but it's not a very nice UX for us to recommend users changing non-public cluster settings so we should fix this as soon as we can